packages/react-components/react-tags-preview/src/components/InteractionTagPrimary/useInteractionTagPrimaryStyles.styles.ts

```diff
@@ -185,6 +184,56 @@ const useRootWithSecondaryActionStyles = makeStyles({
   },
 });
 
+/**
+ * Styles for root slot under windows high contrast mode when Tag is with secondary text.
+ * Tag's primary text has negative margin that covers the border. Pseudo element is used to draw the border.
+ */
+const useRootWithSecondaryTextContrastStyles = makeStyles({
+  rounded: {
+    '@media (forced-colors: active)': {
+      position: 'relative',
+      '::before': {
+        content: '""',
+        ...shorthands.borderTop(tokens.strokeWidthThin, 'solid'),
+        position: 'absolute',
+        top: '-1px',
+        left: '-1px',
+        right: '-1px',
+        bottom: '-1px',
```

Comment:
An amount of classes that will be generated really worries me.

Can we move something to base styles under `makeResetStyles` call?  For example this:

```
'@media (forced-colors: active)': {
      position: 'relative',
      '::before': {
        content: '""',
        ...shorthands.borderTop(tokens.strokeWidthThin, 'solid'),
        position: 'absolute',
        top: '-1px',
        left: '-1px',
        right: '-1px',
        bottom: '-1px',
```

Reply:
Oh I wanted to update you that I changed to use different makeResetStyles for different shape (rounded/circular), but I saw you're faster and approved. Thx 🙏